packages/sdk/src/errors/error-info.ts

```diff
@@ -0,0 +1,34 @@
+import {BigNumberish} from 'ethers'
+
+export enum HolographErrorCode {
```

Comment:
Are we sure error codes is really the best pattern? Generally, I am not a fan as error codes don't really have any inherent meaning. I think a pattern that uses named errors would be more intuitive and preferable, but that's just me